include/alpaka/acc/AccGpuUniformCudaHipRt.hpp

```diff
@@ -41,7 +41,10 @@
 namespace alpaka
 {
     template<typename TApi, typename TAcc, typename TDim, typename TIdx, typename TKernelFnObj, typename... TArgs>
-    class TaskKernelGpuUniformCudaHipRt;
+    class TaskKernelNonCooperativeGpuUniformCudaHipRt;
+
+    template<typename TApi, typename TAcc, typename TDim, typename TIdx, typename TKernelFnObj, typename... TArgs>
+    class TaskKernelCooperativeGpuUniformCudaHipRt;
```

Comment:
and use `CooperativeKernel` instead of `KernelCooperative` for the new functionality.